metaflow/datastore/s3.py

```diff
@@ -7,21 +7,27 @@
 import sys
 import json
 import gzip
+from io import BytesIO
 
 try:
     # python2
     from urlparse import urlparse
-    import cStringIO
-    BytesIO = cStringIO.StringIO
 except:
     # python3
     from urllib.parse import urlparse
-    import io
-    BytesIO = io.BytesIO
 
 from .datastore import MetaflowDataStore, DataException, only_if_not_done
 from ..metadata import MetaDatum
 from .util.s3util import aws_retry, get_s3_client
+from .. import metaflow_config
```

Comment:
Can we move this above L:20?

While on this subject do you know if there is a special format that python auto-format tools follow for organizing import statements (alphabetical, relative references last etc.)?

I usually follow alphabetical, and prefer absolute imports first then relative (like `from .metadata import foo`)

Reply:
There is usually a preference. I think it is system ones first then user ones. Both in alphabetical order. Not sure about absolute vs relatives. I think once we run the auto format tools, this should reformat everything appropriately but can definitely move this one above if you want.